.github/workflows/test.yaml

```diff
@@ -17,9 +20,17 @@ jobs:
               uses: actions/setup-node@v4
               with:
                   node-version: 18.x
+            - name: NPM Registry Auth
+              run:
+                  echo "//npm.pkg.github.com/:_authToken=${{ env.GITHUB_TOKEN
+                  }}" > .npmrc
+              env:
+                  GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
My bad I forgot to do this here for making sure the CI would still work. This is good though it should just work

Reply:
Gas 🔥